### ISSUE TITLE
Website feature request: multiple features

### ISSUE THREAD

**Issue by nicooprat** [Fyrd/caniuse#2318]
Didn't found any info, I think this feature hasn't been proposed yet. The idea is to be able to have more than one feature when presenting browser support.

Here are my ideas:
* URL could be something like `#feat=es5+fileapi`.
* between the search box and the results list, there's a little tag box with the current feature selected (corresponding to the one in the URL) ; there could be more than one, with the ability to remove one at a time.
* on the left side of each result, there's currently a `#` link ; there could also be a `+` to add the specified feature to the tag list (or `-` to remove it).
* browser support would be calculated in a `&&' way: each browser must support all the requested features to be listed as supported.

Typical use case: I'm writing a javascript plugin or anything, I know I'm using these particular features (eg ES5 and the File API), I want to know the global browser support that I can write in the Github repo Readme.

Dont know if I'm clear :) Hope it makes sense!

**Comment by Fyrd** [comment]
This actually already exists, without the +/- buttons:
http://caniuse.com/#feat=es5,fileapi

**Comment by nicooprat** [comment]
Hmm I didn't try the comma separator :) Nice!

Ok so technically it's working but there's no interface for it? Why is that?

**Comment by Fyrd** [comment]
Because making it technically work from URL was much easier than figuring out a decent UI for it and making it work. If I were to go with your suggestion right now it wouldn't work because the moment you "add" a feature from a search, it would update the URL and just display that feature instead of search results...then doing a new search would erase your list (right now you can have just one or the other). So yeah...needs more thought.

**Comment by nicooprat** [comment]
Right, didn't thought about this. We could have something like a "basket", where we could add features (stored in localStorage, or anything but URL) and then click the link to the "multiple-features-summary".

**Comment by Smiley43210** [comment]
If a UI design for this idea is still being explored, you could add a button to each "feature support box", that, when clicked adds that browser feature to a "tracked" list. It could show up as tracked by displaying it as a short row just below the search bar that contains the title of the browser feature. The statistics at the bottom of the page can toggle between "Features shown on page" or "Tracked features". That way, this "tracked" terminology closely follows the "browser tracking" that you can configure in the settings.

**Comment by Fyrd** [comment]
The beta site (https://beta.caniuse.com) now has controls for this, you can star features to add them to a saved list, which includes a "Feature summary" table with multiple views for the combined support data.

Will close this issue once it's on the main site.

**Comment by timdream** [comment]
@Fyrd Good to know! Looking forward to see the beta ship!

However, it takes one more click than I like to get to the link to share — one would have to click on the star to get to `/ciu/saved` to find the "Link to share" link.

Is it possible to show the sharable link with, say, a 🔗 button besides the ⭐️ button? It can be labeled as "Permalink to saved features."

Thanks!

**Comment by timdream** [comment]
Also, there is an anchor link to "feature summary" in the saved features page, but there isn't one for the multiple feature list page, even though I can see the summary when I scroll down the page.

**Comment by SlimSim** [comment]
The FileSystemAPI is not found on canIuse, i think that would be a great adition!

**Comment by yume-chan** [comment]
This feature doesn't work correctly with multiple MDN features.

For example, https://caniuse.com/mdn-api_readablestream,mdn-api_websocketstream shows "ReadableStream API" twice, instead of "ReadableStream API" and "WebSocketStream API"
